src/shared/utils/formatters.ts

```diff
@@ -9,16 +9,15 @@ export function getFormattedCandidateName(candidateName: ICandidateName): string
 
 /**
  * `Builds a formatted html structure from a given IAddress
- * todo - this is quick and dirty for now. Needs to deal with missing fields etc
```

Comment:
Can we keep the comment that this is quick and dirty? We should really avoid having markup in .ts files. HTML belongs in .html files. Think this was something done in haste and should be discouraged.

Reply:
ok -done